lib/descriptor/toolchain.rs

```diff
@@ -21,7 +21,7 @@ impl Toolchain {
         Get the toolchain of the current host system.
     */
     #[must_use]
-    pub fn current_system() -> Option<Self> {
+    pub const fn current_system() -> Option<Self> {
```

Comment:
Feels a bit presumptuous to make this `const` considering it's not correctly implemented yet.

Reply:
Agreed. Went ahead and implemented it now, still `const` as predicted 😎